service/web/tabs/app/package.json

```diff
@@ -15,21 +15,21 @@
     "prebuild": "npm run-script lint",
     "reinstall": "rm -rf node_modules && npm run-script install",
     "start": "npm run-script prebuild && cross-env NODE_ENV=development webpack-dev-server",
-    "test": "jest --passWithNoTests --maxWorkers=2",
```

Comment:
Kinda weird this changed, try `miskweb prebuild -e`

Reply:
I already did that